Update reference documentation (add data model and GeoPackage) (#310)
Open-Telecoms-Data/open-fibre-data-standard: docs/reference/publication_formats/json/index.md

<!-- docs-type: reference https://diataxis.fr/reference/ -->

# JSON

> This section describes how to represent the [OFDS data model](../../data_model.md) as [JSON](https://www.json.org) data. It describes the [JSON Schema for a network object](network_schema.md), and the [containers](containers.md) for publishing, storing or exchanging one or more network objects.

Reviewer: ```suggestion
This section describes how to publish [OFDS conformant data](../../data_model.md) in [JSON](https://www.json.org) format. It describes the [JSON Schema for a network object](network_schema.md), and the [containers](containers.md) for publishing, storing or exchanging one or more network objects.
```

Author: I'll update once we agree on wording in https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/310#discussion_r2702842568

Author: Updating to reflect the wording used on the GeoPackage page:

```suggestion
This page describes the schema for publishing, storing or exchanging data that conforms to the [OFDS data model](../../data_model.md)  as [JSON](https://www.json.org) data. It describes the [JSON Schema for a network object](network_schema.md), and the [containers](containers.md) for publishing, storing or exchanging one or more network objects.
```
